packages/cardpay-sdk/package.json

```diff
@@ -1,10 +1,17 @@
 {
   "name": "@cardstack/cardpay-sdk",
   "description": "An SDK for using the cardpay protocol",
-  "version": "0.28.2",
+  "version": "0.28.12",
   "author": "Hassan Abdel-Rahman @habdelra",
-  "main": "./dist/index.js",
+  "main": "./dist/esm/index.js",
   "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "require": "./dist/index.js",
+      "import": "./dist/esm/index.js",
+      "types": "./dist/index.d.ts"
+    }
+  },
```

Comment:
this is the new shiny thing that node recognizes that will allow the CLI to use the CJS entrypoint. Metro hopefully can understand this, otherwise, it should be able to fallback to the `main` property which defaults to the ESM entrypoint.

Reply:
Arg--it looks like TS is mad now, because it's trying to use ./dist/esm/index.js. i mean we can allow that to happen, but that's kind of annoying because it's yet another build you have to have running. i wonder if there is a rule that will work for TS but not for metro...

Reply:
According to this, metro does not understand package.json `exports`, so this approach probably works in our favor https://github.com/facebook/metro/issues/670